### PR TITLE
Add pdc-client doc.

### PR DESCRIPTION
**Pull request by zkl94**
JIRA： PDC-1211

### CODE REVIEW

File: docs/pdc_client.rst [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+For Developers
+--------------
+
+Instalation details
+~~~~~~~~~~~~~~~~~~~
+
+#. yum repository
+
+   If you have installed PDC Server by some yum repository, PDC Client is in
```
Comment (xychu) [inline docs/pdc_client.rst]: This part need updates, since no other docs to refer.
Same as L#158

**Comment by lubomir** [comment]
This should add the full sphinx setup with `Makefile`. The document makes use of `sphinx.ext.autodoc` (line 95 and below), and reading that in rst format is not really nice.

Generating and deploying the documentation to github pages would be nice, but probably not a hard requirement.

**Comment by zkl94** [comment]
@lubomir yes, I will fix it.

**Comment by xychu** [comment]
One new pr #14 to include client doc into README, if no more docs(sphinx) needed, let's close this.

**Comment by xychu** [comment]
PR #14 merged.

**Pull request closed without merging**